addons/web_editor/static/src/js/editor/odoo-editor/test/spec/link.test.js

```diff
@@ -93,6 +93,7 @@ describe('Link', () => {
         testNotUrlRegex('a.b.bc');
         testNotUrlRegex('20.08.2022');
         testNotUrlRegex('31.12');
+        testUrlRegex('https://github.com/repo/compare/branch-1.2...branch-dev:repo:branch-1.2-branch-name');
```

Comment:
Could you please simplify the test URL to highlight what is being tested ( `/abc...def`).


Could you also, if possible, add other test url that highlight the changes you make of the regex url.
We want to have the best coverage possible on this regex.